code/game/gamemodes/game_mode.dm

```diff
@@ -246,12 +240,47 @@
 							//			recommended_enemies if the number of people with that role set to yes is less than recomended_enemies,
 							//			Less if there are not enough valid players in the game entirely to make recommended_enemies.
 
+// Just the above proc but for alive players
+/datum/game_mode/proc/get_alive_players_for_role(role, override_jobbans = FALSE)
+	var/list/players = list()
+	var/list/candidates = list()
+
+	var/roletext = get_roletext(role)
+
+	// Assemble a list of active players without jobbans.
+	for(var/mob/living/carbon/human/player in GLOB.player_list)
+		if(player.client && !(locate(player) in SSafk.afk_players))
+			if(!jobban_isbanned(player, ROLE_SYNDICATE) && !jobban_isbanned(player, roletext))
+//				if(player_old_enough_antag(player.client, role))
```

Comment:
is this even still relevant code? If not I'd get rid of the comment

Reply:
Removed!